ansible/files/bootstrap/telemetry.sh

```diff
@@ -13,6 +13,17 @@ then
     IS_MASTER=$(jq .isMaster /mnt/var/lib/info/instance.json)
 fi
 
+# Pull information about the instance
+CLUSTER_ID=$(jq -r .jobFlowId /mnt/var/lib/info/job-flow.json)
+CLUSTER_NAME=$(describe_cluster '.Cluster.Name')
+
+# Cache results of a call to describe-cluster, and provide a query interface
+aws emr describe-cluster --region us-west-2 --cluster-id "$CLUSTER_ID" > describe-cluster.json
+describe_cluster() {
+    local query=$1
+    jq -r "$query" describe-cluster.json
+}
```

Comment:
We only make a single describe-cluster call now.

Reply:
nice